site/pages/create-consequence-network/create-consequence-network.test.tsx

```diff
@@ -18,6 +18,17 @@ const withInputs: CreateConsequenceNetworkProps = {
     },
 };
 
+beforeAll(() => {
+    vi.mock("next/router", () => ({
+        useRouter() {
+            return {
+                pathname: "",
+                query: "",
+            };
+        },
+    }));
+});
+
 describe("pages", () => {
```

Comment:
Do we need a test in here to show it renders the cancel button if the conditions are met?

Reply:
do you want to check if the screen is rendered when the useRouter mock returns a value for query attribute?

Reply:
There is logic in the page that means we render it differently depending on the router query - we render the Cancel changes button, or we dont render it. We need a test to check that the page is correctly doing this.

Reply:
done